bin/install-site.sh

```diff
@@ -307,6 +323,15 @@ Pin-Priority: 50
 EOF
     fi
 
+    if [ x"$DISTRIBUTION" = x"debian" ] && [ x"$DISTVERSION" = x"jessie" ]
```

Comment:
Is this being added because it's definitely needed? What packages are required; I'm guessing pdftk from the list, anything else? Should this actually be using wheezy at least; looks like pdftk, yui-compressor, and libunix-mknod-perl are the only things in the Debian squeeze repo, all seem to be in Debian wheezy. squeeze is only LTS for 10 more days :)